sda/local.go

```diff
@@ -398,6 +398,11 @@ func (l *LocalTest) NewLocalConode(port int) *Conode {
 
 }
 
+// NewContext creates a context from a ServerIdentity.
+func (l *LocalTest) NewContext(si *network.ServerIdentity, servID ServiceID, manager *serviceManager) *Context {
```

Comment:
Removed the `NewContext` altogether and using an interface in the `NewPropagationFunc` now, which is probably cleaner anyway.